draft-ietf-quic-transport.md

```diff
@@ -92,9 +92,12 @@ informative:
 
 --- abstract
 
-This document defines the core of the QUIC transport protocol.  Accompanying
-documents describe QUIC's loss detection and congestion control and the use of
-TLS for key negotiation.
+This document defines the core of the QUIC transport protocol. QUIC provides
+streams for structuring application data, flow control,
```

Comment:
structuring application data is not clear to me.

```suggestion
flow-controlled streams for transmitting application data,
```

Reply:
I don't think you want to remove flow control from the connection-level descriptors of QUIC.

On "structuring data", the idea is that streams are a way to retain and structure application data. How about "QUIC provides streams for retaining structure in application data, flow control" ?

Reply:
Streams are flow controlled, but other frames are not, so I was attempting to connect the two.  Saying you have flow control and not indicating what you're controlling seems potentially confusing to me.

Reply:
I think that I prefer Ian's suggestion here.  Breaking these apart doesn't make sense any more.

This could be two sentences instead: one for streams, one for the prime characteristics.  But then Ian's suggestion is shorter than my original proposal.

Reply:
> QUIC provides applications with flow-controlled streams for structured communication, low-latency connection establishment, and network path migration.  QUIC includes security measures that ensure confidentiality, integrity, and availability in a range of deployment circumstances.

^^ ?

Reply:
I like your proposed text @martinthomson